Kvent/templates/kvent/event-detail.html

```diff
@@ -17,11 +17,16 @@
     </nav>
     <div class="row">
         <hr class="line">
+        <div class="event-image">
+            <img src="{{ event.get_photo.url }}" alt="sad" width="300px" height="auto" />
```

Comment:
what is **/** after height attribute?

Reply:
I use "/" because I'm to lazy to write <img></img>